docs/sql-ref-syntax-ddl-create-view.md

```diff
@@ -61,6 +61,17 @@ CREATE [ OR REPLACE ] [ [ GLOBAL ] TEMPORARY ] VIEW [ IF NOT EXISTS ] view_ident
     * `[ ( column_name [ COMMENT column_comment ], ... ) ]` to specify column-level comments.
     * `[ COMMENT view_comment ]` to specify view-level comments.
     * `[ TBLPROPERTIES ( property_name = property_value [ , ... ] ) ]` to add metadata key-value pairs.
+    * `[ WITH SCHEMA { BINDING | COMPENSATION | [ TYPE ] EVOLUTION } ]` to specify how the view reacts to schema changes
+
+      This clause is not supported for `TEMPORARY` views.
```

Comment:
Temp view and persistent view share the same view resolution code path, what should be the mode for temp view? Still BINDING as before?

Reply:
Temporary views are governed by the config. If the feature is disabled its unchanged (BINDING ). Otherwise COMPENSATION (our new default).